lib/Model/Importer/Binary/PECOFFImporter.cpp

```diff
@@ -233,6 +233,10 @@ void PECOFFImporter::parseImportedSymbols() {
       continue;
     }
 
+    if (Model->ImportedLibraries.count(Name.str()))
+      continue;
```

Comment:
I think this is redundant: `SortedVector::insert` already does this. There are more occurrences of this.

Reply:
But it prevents us to redundantly call the functions bellow (used for generating entries for dynamic functions of our Model) as well.

Reply:
Ah OK, consider:
```suggestion
    if (not Model->ImportedLibraries.insert(Name.str()).second)
      continue;
```